tile_demo.html

```diff
@@ -0,0 +1,86 @@
+
```

Comment:
This file needs to be removed.

Reply:
Yeah, I've noticed it. I kind of assumed that these generated files are in gitignore.